components/client/src/main/java/com/hotels/styx/client/connectionpool/SimpleConnectionPool.java

```diff
@@ -187,6 +183,13 @@ public void connectionClosed(Connection connection) {
         return this.stats;
     }
 
+    private double utilisation() {
+        double usage = borrowedCount.get() + waitingSubscribers.size();
+        double limit = poolSettings.maxConnectionsPerHost() + poolSettings.maxPendingConnectionsPerHost();
```

Comment:
is this correct? I would think the limit is "maxConnectionsPerHost", so if we allow 100 connections and we have 100 established connections, the pool is completely used.

Usage would probably be also borrowedCount? I don't know if it makes sense that we say that we have a pool usage of 120%. Our usage is 100% and then there's a few connections waiting to be served, those will fail if  borrowedCount does not go down...  

In general, is this utilisation metric required if we are publishing all the others?

Reply:
It would be more sensible to publish the pool size (max active connections, max pending subscribers) as metrics, and calculate the utilisation in the dashboards.